src/Symfony/Component/Validator/ValidatorBuilderInterface.php

```diff
@@ -117,13 +116,6 @@ public function disableAnnotationMapping();
      */
     public function setMetadataFactory(MetadataFactoryInterface $metadataFactory);
 
-    /**
-     * Sets the cache for caching class metadata.
-     *
-     * @return $this
-     */
-    public function setMetadataCache(CacheInterface $cache);
```

Comment:
no need to remove this one, isn't it?

Reply:
It is not mandatory anymore to implement that method. But yes, since the interface will go away with 5.0 anyway, I might as well leave it as it is. Shall I revert?

Reply:
yes please, it's unrelated

Reply:
Done.